chain/filecoin/account.go

```diff
@@ -0,0 +1,294 @@
+package filecoin
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+
+	filaddress "github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-jsonrpc"
+	"github.com/filecoin-project/lotus/api"
+	filclient "github.com/filecoin-project/lotus/api/client"
+	"github.com/filecoin-project/lotus/chain/types"
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	"github.com/filecoin-project/specs-actors/actors/abi/big"
+	"github.com/filecoin-project/specs-actors/actors/crypto"
+	"github.com/ipfs/go-cid"
+	"github.com/minio/blake2b-simd"
+	"github.com/renproject/multichain/api/account"
+	"github.com/renproject/multichain/api/address"
+	"github.com/renproject/multichain/api/contract"
+	"github.com/renproject/pack"
+)
+
+const (
+	// AuthorizationKey is the header key used for authorization
+	AuthorizationKey = "Authorization"
+
+	// DefaultClientMultiAddress is the RPC websocket URL used by default, to
+	// interact with the filecoin lotus node.
+	DefaultClientMultiAddress = "ws://127.0.0.1:1234/rpc/v0"
```

Comment:
This is not a valid multi-address? Given that, I am surprised that this actually works. Consider changing to an actual multi-address, or consider renaming the variable (if it is not actually required to be a multi-address).

Reply:
I think we can simply call this `DefaultClientRpcUrl`.